kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/rest/OrganizationsApiImpl.java

```diff
@@ -127,5 +224,18 @@ public Response listServiceClasses(String organizationIdParam) {
     
     return Collections.unmodifiableList(result);
   }
+  
+  private List<EventProvider> getEventProviders() {
+    // TODO: Prioritize providers
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)